### ISSUE TITLE
AttributeError: module 'jax' has no attribute 'scipy'

### ISSUE THREAD

**Issue by anny0316** [google/jax#10222]
Hello，my python version=3.6,  I have installed jax-0.2.22 and jaxlib-0.1.69.
How to deal with the problem? thanks a lot.

**Comment by YouJiacheng** [comment]
Can you upgrade to jax==0.3.5 ?

**Comment by jakevdp** [comment]
I'm not sure what might have caused this; I tried running this with the versions you have installed and it all seems to work:
```python
import jax
import jaxlib

print(jax.__version__)
# 0.2.22
print(jaxlib.__version__)
# 0.1.69

from jax import scipy
print(scipy)
# <module 'jax.scipy' from '/usr/local/lib/python3.7/dist-packages/jax/scipy/__init__.py'>
```
Can you include a minimal reproducible example of the code that led to the error? Thanks!

**Comment by jakevdp** [comment]
Again, the error message alone is not enough to help you. Please include the code you ran (a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible) so that we can reproduce the error.

**Comment by anny0316** [comment]
I am sorry for that. Now my environment is: python=3.7, jax=0.3.5, jaxlib=0.3.2.
I have a program as follows:
@partial(jax.jit, backend='cpu')
def get_trajectory_analytic(y0, times, **kwargs):
    return odeint(analytical_fn, y0, t=times, rtol=1e-10, atol=1e-10, **kwargs)

When run the program, the fofllowing error appear:
TypeError: odeint() got an unexpected keyword argument 'mxsteps'.

It is strange that the odeint function in jax.experimental.ode has the mxstep parameter, not the mxsteps parameter.

**Comment by jakevdp** [comment]
Thank you for including the function that is causing the issue, but a complete example would also include how the function is called. At this point, my buest guess is that you are passing a keyword argument named `mxsteps` when you call the function, and you could fix the error by passing `mxstep` instead.

**Comment by anny0316** [comment]
Yes, you are right. The original expression: vget = partial(jax.jit, backend='cpu')(jax.vmap(partial(get_trajectory_analytic, mxsteps=100), (0, None), 0)).  I changed the mxsteps parameter to mxstep.
However, the following error occur:
Traceback (most recent call last):
  File "/home/lii/.conda/envs/py37/lib/python3.7/site-packages/jax/_src/api_util.py", line 171, in argnums_partial_except
    hash(static_arg)
TypeError: unhashable type: 'DynamicJaxprTracer'

**Comment by hawkinsp** [comment]
Closing because we never got a runnable reproduction of the problem. If you have a complete runnable reproduction of the problem we'd be delighted to help out!

**Comment by azh2000xy** [comment]
i meet the same problem as you did，i guess you are working on the lnn problem which is writen by cranmer，if you solve this problem now，i hope you can tell me，i will appreciate it

**Comment by lwkobe** [comment]
> 
I also meet this problem, How to fix it?
